### ISSUE TITLE
USWDS - a11y: Ensure scope attribute is used correctly on tables.

### ISSUE THREAD

**Issue by mejiaj** [uswds/uswds-site#1110]
## Description

Related to #1104.

**id:** 'scope-attr-valid',
**impact:** 'moderate',
**tags:** [ 'cat.tables', 'best-practice' ],
**description:** 'Ensures the scope attribute is used correctly on tables',
**help:** 'scope attribute should be used correctly',
**helpUrl:** 'https://dequeuniversity.com/rules/axe/3.5/scope-attr-valid?application=axeAPI',

### Pages to fix
- [ ] accordion
- [ ] alert
- [ ] banner
- [ ] breadcrumb
- [ ] footer
- [ ] form-controls
- [ ] header
- [ ] identifier
- [ ] search
- [ ] sidenav
- [ ] step indicator

## Steps to reproduce the issue

1. Open `run-axe.js`
2. Add component URLs from #1104 to `const PAGES` array
4. `npm run axe`

**Comment by mejiaj** [comment]
Blocked by #1136

**Comment by thisisdano** [comment]
I'll be sure to add this to the work in #1136
